docs/dashboards/chart-builder.md

```diff
@@ -0,0 +1,176 @@
+than one metric is picked you have to apply transformations to let Chart Builder know what is
+the relation between metrics. If you don't write your own
+expression in transformation section all metrics are added to each other producing
+a single series.
+
+## Transformation
+
+Transformations are used to modify metrics with a formula. Why is this useful? Imagine you collect
+all the ingredients of cpu usage separately, like `os.cpu.user`, `os.cpu.system`, `os.cpu.wait`,
+... If you want to chart all the ingredients as a single series you have to
+sum them in an expression like the one below
+
+```
+`os.cpu.user` + `os.cpu.system` + `os.cpu.wait`
+```
+
+Another example is ratio. Imagine that you are interested in the ratio of the network input
+traffic and the network output traffic so you can easily spot suspicious behaviors of the system
+(sudden jump of output traffic which is not correlated with the input traffic jump). Then
+you simply divide one metric by another to get the interesting series:
+
+```
+`os.network.out` / `os.network.in`
+```
+
+_NOTE: All the metrics which have dot in the name have to be mentioned between_ ``` `` ``` .
+
+### Math operators
+
+You already know why transformations are needed. Here is what you can use to build transformations.
+
+We support math operators `+`, `-`, `*`, `/`, `%`. Parts of the expressions can be scoped
+with parenthesis `()`. For example
+
+```
+`os.cpu.user` / (`os.cpu.user` + `os.cpu.system` + `os.cpu.wait`) 
+```
+
+### Functions
+
+In addition to math operations there are functions which also can be used. The list of functions
+will be growing.
+
+##### ifNull
+
+  Returns an alternative value if the main argument is NULL.
+
+  __Usage__:
+
+  ```ifNull(x, y)```
+
+  __Example__:
+
+  ```ifNull(`os.disk.usage`, 0)```
+
+
+##### coalesce
+
+  Checks from left to right whether NULL arguments were passed and returns the first
+  non-NULL argument. This function gets any number of parameters.
+
+
+  __Usage__:
+
+  ```coalesce(x, y, ...)```
+
+  __Example__:
+  
+  Consider you collect 2 different metrics which have similar meaning. On one host you collect a
+  metric which is called `os.disk.usage` and on the second host you collect a metric called
+  `os.disk.used`. You'd like to show a total disk usage from 2 hosts. Unfortunately metrics
+  are called differently. Here's where `coalesce` is useful. You simply call the function 
+  ```coalesce(`os.disk.usage`, `os.disk.used`, 0)```. Depending on the metrics' name for a given
+  datapoint one of `os.disk.usage`, `os.disk.used` and `0` will be picked.
+
+
+## Filter by
+
+Filter section is used to specify a criteria that is used to find only matching datapoints.
+Suppose you collect data from multiple hosts but you'd like to see a chart containing only
+2 of them (`host1` and `host2`). You have to pick the right filter
+in [Filter by](#filterby) section. You chose tag name `os.host` and 2 tag values `host1` and `host2`.
```

Comment:
in the [Filter by](#filterby) section. You chose tag name `os.host` and 2 tag values `host1` and `host2`.